### PR TITLE
Fixed Tab group header to be scrollable 

### LINKED ISSUE #198: [Bug]: Implement scrollable Tab Header
### Components Version

0.9.1

### Which package type are you using?

Bundled for ESM

### What type of bug is this?

Rendering Error

### Test Case/Reproduction

_No response_

### Description

Issue reported on project leveraging the Tab related components.

By default when using the tab related components in this case using a tab-group with nested tabs this will generate tab headers.

When the amount of tabs causes some of the tab headers to be rendered off-screen we require the ability to scroll the specific tab header into view so that tab can be selected.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have [searched for existing issues](https://github.com/capitec/omni-components/issues) and to the best of my knowledge this is not a duplicate

### PR DESCRIPTION
**Pull request by chromaticWaster**
### Description:

closes #198 

Issue raised by project using Omni-components when rendering the Tab related components in a containing element with a limited width the tab headers render off screen. 

The component needs to introduce the ability to scroll the tab headers to allow interaction with tabs that are not in view.

### Screenshots (if appropriate):

Issue demonstrated on the doc site, for the purposes of this test we limited the max width of the interactive story to have a max-width of 400px and have 12 nested tabs;

**Before** 

image

**After**

image

### All Submissions:

* [x] I have followed the [contribution guidelines](https://github.com/capitec/omni-components/blob/develop/CONTRIBUTING.md) for this project.
* [x] I have checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change.

### Changes to Core Features:

* [x] I have added an explanation of what my changes do and why I would like to include them.
* [x] I have written new tests for my core changes, as applicable.
* [x] I have successfully ran tests with my changes locally.
* [x] I have added/updated docs, as applicable.

### CODE REVIEW

File: src/tab/TabGroup.ts [hunk 1]
```diff
@@ -150,6 +150,8 @@ export class TabGroup extends OmniElement {
                     display: flex;
                     flex-direction: row;
                     align-items: center;
+                    overflow-x: auto;
```
Comment (stefan505) [inline src/tab/TabGroup.ts:153]: Please make a CSS var for this.

File: src/tab/TabGroup.ts [hunk 2]
```diff
@@ -150,6 +150,8 @@ export class TabGroup extends OmniElement {
...
+                    overflow-y: hidden;
```
Comment (stefan505) [inline src/tab/TabGroup.ts:154]: Please make a CSS var for this.

**Review by stefan505 — CHANGES_REQUESTED** [review]

File: src/tab/TabGroup.ts (hunk 2 above)
Comment (chromaticWaster) [reply src/tab/TabGroup.ts]: Added accordingly with update to JS doc comment.

File: src/tab/TabGroup.ts (hunk 1 above)
Comment (chromaticWaster) [reply src/tab/TabGroup.ts]: Added accordingly with update to JS doc comment.

**Review by stefan505 — APPROVED** [review]

**Review by BOTLANNER — APPROVED** [review]

**Pull request merged**